esmvaltool/recipes/ref/recipe_ref_scatterplot.yml

```diff
@@ -0,0 +1,317 @@
+# ESMValTool
+---
+documentation:
+  title: Model evaluation with focus on clouds.
+  description: >
+    Plot 2D histograms using the Seaborn diagnostic in ESMValTool.
+  authors:
+    - bock_lisa
+    - lauer_axel
+  maintainer:
+    - bock_lisa
+
+timerange_for_data: &time_period
+  timerange: '2007/2011'  # can be specified, this is just an example
+regridding: &regridding
+  regrid:
+    target_grid: 1x1
+    scheme:
+      reference: esmf_regrid.schemes:ESMFAreaWeighted
+levels_for_3d_vars: &extracting_levels
+  extract_levels:
+    levels: {cmor_table: CMIP6, coordinate: plev27}
+    coordinate: air_pressure
+    scheme: linear
+
+
+preprocessors:
+  common_grid_clt:
+    <<: *regridding
+    custom_order: true
+    mask_below_threshold:
+      threshold: 0.001
+    mask_above_threshold:
+      threshold: 100.
+  common_grid_clivi_clwvi:
+    custom_order: true
+    <<: *regridding
+    mask_below_threshold:
+      threshold: 0.001
+    mask_above_threshold:
+      threshold: 0.2
+  common_grid_lwcre:
+    custom_order: true
+    <<: *regridding
+    mask_below_threshold:
+      threshold: -10.
+    mask_above_threshold:
+      threshold: 100.
+  common_grid_swcre:
+    custom_order: true
+    <<: *regridding
+    mask_below_threshold:
+      threshold: -200.
+    mask_above_threshold:
+      threshold: -0.001
+  common_grid_pr:
+    custom_order: true
+    <<: *regridding
+    convert_units:
+      units: mm day-1
+    mask_below_threshold:
+      threshold: 0.00001
+    mask_above_threshold:
+      threshold: 8
+  common_grid_levels_ta:
+    custom_order: true
+    <<: *extracting_levels
+    <<: *regridding
+    mask_below_threshold:
+      threshold: 180
+    mask_above_threshold:
+      threshold: 320
+  common_grid_levels_cli:
+    custom_order: true
+    <<: *extracting_levels
+    <<: *regridding
+    convert_units:
+      units: g kg-1
+    mask_below_threshold:
+      threshold: 0.000001
+    mask_above_threshold:
+      threshold: 0.005
+
+
+diagnostics:
+  plot_joint_clt_swcre_model:
+    description: Scatterplot clt vs swcre.
+    variables:
+      clt: &var_clt
+        mip: Amon
+        exp: historical
+        ensemble: r1i1p1
+        preprocessor: common_grid_clt
+        <<: *time_period
+      swcre: &var_swcre
+        <<: *var_clt
+        preprocessor: common_grid_swcre
+        derive: true
+        force_derivation: false
+    additional_datasets:
+      - {dataset: CESM2, project: CMIP6, grid: gn, ensemble: r1i1p1f1, alias: CESM2}
+    scripts:
+      plot: &plot_settings_jointplot_clt_swcre
+        script: seaborn_diag.py
+        seaborn_func: jointplot
+        title:
+        seaborn_kwargs:
+          kind: hist
+          x: clt
+          y: swcre
+          bins: 20
+          vmax: 0.015
+          stat: probability
+          cmap: YlGnBu
+          cbar: true
+          cbar_kws:
+            extend: 'max'
+          marginal_ticks: true
```

Comment:
I would consider disabling this since this overlaps with the ticks from the main plot:

![seaborn_jointplot](https://github.com/user-attachments/assets/db81cb62-1cff-48f4-b98e-e7c551631265)

Not sure how useful those ticks are in practice. An alternative would be to decrease the fontsize, which should work in the recipe with something like

```yml
seaborn_settings:
  rc:
    xtick.labelsize: ...
```

Reply:
I disable for now the marginal ticks.